src/app/modules/item/components/permissions-edit-dialog/permissions-edit-dialog.component.ts

```diff
@@ -32,10 +36,10 @@ export class PermissionsEditDialogComponent implements OnChanges, OnDestroy {
   };
 
   form = this.fb.group({
-    canView: [ 'none' ],
-    canGrantView: [ 'none' ],
-    canWatch: [ 'none' ],
-    canEdit: [ 'none' ],
+    canView: [ ITEMVIEWPERM_NONE ],
+    canGrantView: [ ITEMGRANTVIEWPERM_NONE ],
+    canWatch: [ ITEMWATCHPERM_NONE ],
+    canEdit: [ ITEMEDITPERM_NONE ],
```

Comment:
If we have a `_MAX` approach upper, shouldn't we have a `_MIN` approach here?
Unless the lowest level is always `'none'`

Reply:
yeah I assume the min was quite straightforward (cannot have a new lower perm).. while max is not and may still change in the future